docs/samples/calling/app.js

```diff
@@ -95,6 +95,16 @@ const devicesById = {};
 const img = new Image();
 const vmAvatarImg = new Image(100);
 
+document.querySelectorAll('.collapsible').forEach((el) => {
```

Comment:
Where are we using tghs ?

Reply:
All the sections are collapsible now. Try clicking on the section headers like `Authentication`, `Media Streams`, `Call initialization` etc except for the first one `Calling SDK`